examples/linear_model/plot_ols_3d.py

```diff
@@ -29,12 +31,15 @@
 y_train = y[:-20]
 y_test = y[-20:]
 
+# %%
+# Next we fit a linear regression model.
+
 ols = linear_model.LinearRegression()
 ols.fit(X_train, y_train)
```

Comment:
I think the only thing I am not sure about is the output of `.fit`, I feel the output is not really interesting:

![image](https://user-images.githubusercontent.com/1680079/155120749-e28437e6-42d9-4805-9fc2-dc3332c874af.png)

so I would probably use the following code even if it is a bit more complicated:
```suggestion
_ = ols.fit(X_train, y_train)
```